content/60.methods.md

```diff
@@ -54,87 +105,35 @@ For instance, the template variable {% raw %}`{{ebm_trials_results}}`{% endraw %
-
-##### Methods
-
-Reviewers began by describing the study question(s) being investigated by the article.
-They then described the study population, the sample size, the prevalence of the disease in the study population, countries / regions considered in case of human subjects, demographics of participants, the setting, and any remaining inclusion / exclusion criteria considered.
-
-##### Assignment
-
-Reviewers described how the treatment is assigned, whether it was an interventional or observational study, whether randomization took place, etc.
-
-##### Assessment
-
-###### Outcome Assessment
-
-Reviewers described the outcome that was assessed and evaluated whether it was appropriate given the underlying study question.
-They described whether there were any missing data such as whether there were individuals lost to follow up.
-They then describe whether there were any potential sources of bias such as lack of blinding in a randomized controlled trial.
-
-###### Statistical Methods Assessment
-
-Reviewers described which statistical methods were used for inference and whether applied methods were appropriate for the study.
-They then described whether adjustments were made for possible confounders.
+<!-- To Do: Describe Ryan's work making it possible to cite clinical trial identifiers-->
+<!-- To Do: Also, if we end up automating the Manubot-to-LaTeX workflow while trying to submit this paper, we should describe that too-->
+Due to the needs of this project, project contributors also implemented new features in Manubot and [Zotero](https://www.zotero.org/), which Manubot uses to extract metadata for some types of citations.
+These features support directly citing clinical trial identifiers such as `clinicaltrials:NCT04292899` [@clinicaltrials:NCT04292899] and generating the complete review manuscript along with the individual manuscripts reviewing specific topics. <!-- To Do: reference individual manuscripts -->
 
-##### Results
+<!--To Do: This is old text, separate out into intro what was pre-existing and here what was added over the course of this project-->
+Finally, Manubot and GitHub Actions continuous integration allowed for scripted updates to be run each time the manuscript was generated.
+These scripts were used to check that the manuscript was built correctly, run spellchecking, and cross-reference the manuscripts cited in this review, summarized in Appendix A, and discussed in the project's issues and pull requests.
 
-Reviewers described the estimated association between the treatment and outcome.
-They described measures of confidence or statistical significance, if provided.
+### Results
 
-##### Interpretation
+#### Recruitment and Community Building
 
-Reviewers described whether a causal claim could be made.
-They described whether any side effects or interactions with other drugs were identified, as well as any subgroup findings.
+<!--To Do: Add a graph/text describing number of unique contributors & commits over time-->
```

Comment:
For this type of conference, we should include a Results section in addition to the Methods.  I haven't checked the template, but I assume that will be expected.

Our major claim is that Manubot successfully facilitated a massive collaborative review on an urgent topic.  Our Results section can provide a qualitative and quantitative assessment to support that claim.  I noted above that the growth of the literature could be one initial result to demonstrate the problem (and perhaps the grown of clinical trials, treatments, etc. as well).  Other quantitative results could include:
- unique contributors and commits over time, as noted in the comment
- growth of the manuscript over time (we could try regenerating [this figure](https://greenelab.github.io/meta-review/#fig:contrib) summarizing deep review growth)
- growth of paper issues over time
- distribution of review contents, that is, something showing the number of comments per pull request that was merged
- growth of the reference list over time and/or number of clinical trials cited (because we describe that as a necessary enhancement)
- example of how the auto-updated figures changed between an early version and most recent version to demonstrate the importance of having current data visualized

Is there anything else along these lines we could include in a Results section?  Most of my suggestions are plots showing something getting bigger over time, which is okay but not the most exciting data.

We could have a summary figure showing all of the external data sources that are integrated into the manuscript.  We have icons for MSSM reviews, JHU data, etc. at the top.  That flows into a GitHub repo, which also takes input from all the contributors.  Then the output of the repo is the manuscript and other CI artifacts.